### ISSUE TITLE
Unable to replace colours within functions.

### ISSUE THREAD

**Issue by TomStrepsil** [wuifdesign/postcss-replace-values#7]
Hello again!

I'm not sure if the[ commit to fix css variable matching](https://github.com/wuifdesign/postcss-replace-values/commit/89a12763491ce165df46c564bae6ad693dca0358) introduced this, but we have a regression in our ability to replace css variables within functions, which I think was working as some point before.

I would hope the following test would pass:
```javascript
  it("should support replacing variables within functions", async () => {
    const source = 'div { color: rgba(var(--foo), 0.9); }';
    const checkValue = await process(source, { values: { '--foo': '--bar' } });
    expect(checkValue).toBe('div { color: rgba(var(--bar), 0.9); }');
  });
```
...but I get the result:
```
  Expected: "div { color: rgba(var(--bar), 0.9); }" 
  Received: "div { color: rgba(var(--foo), 0.9); }" 
```
I think modifying [the code](https://github.com/wuifdesign/postcss-replace-values/blob/main/lib/index.ts#L21) to check anywhere _within_ a `replaceValue` makes the test(s) pass:
```javascript
    if (search.slice(0, 2) === '--') {
      return replaceValue === search || replaceValue.includes(`var(${search})`);
    }
```
...and it appears to honour the "not replace part" fix put in place in the original commit:
```javascript
  it('should not replace part of a css variable when used in a function', async () => {
    const source = 'div { color: rgba(var(--foobar), 0.9); }';
    const checkValue = await process(source, { values: { '--foo': '--boz', '--foobar': '--baz' } });
    expect(checkValue).toBe('div { color: rgba(var(--baz), 0.9); }');
  });
```
We can work around this by simply specifying the replace values as:
```json
{ "var(--foo)": "var(--bar)" }
```
..rather than:
```json
{ "--foo": "--bar" }
```
...but doesn't seem in the spirit of the work you've done here.

Once again, thanks for everything on this plugin!  Hope my suggestions are helping improve things, happy to contribute PRs if you'd prefer.

**Comment by wuifdesign** [comment]
fixed with `v2.3.0`

**Comment by TomStrepsil** [comment]
Thanks so much!
